charts/posthog/README.md

```diff
@@ -127,7 +205,7 @@ postgresql:
 
 redis:
   master:
-    size: 2Gi
+    size: 10Gi
```

Comment:
Why?

Reply:
The default in our values is 5Gi, so this going smaller didn't make sense (given that this section is about scaling up further from default values). If we don't need to increase it to scale up I'd just remove this redis size change instead.

Reply:
I don't think we really need to scale this up at all. Redis is treated more as a ephemeral service by everything but plugins.